pudb/debugger.py

```diff
@@ -1954,6 +1955,25 @@ def show_output(w, size, key):
             raw_input("Hit Enter to return:")
             self.screen.start()
 
+        def reload_breakpoints_and_redisplay(w, size, key):
+            bp_source_identifier = \
+                self.source_code_provider.get_source_identifier()
+
+            def set_of_line_nums_from_bp_list(bp_list):
+                result = set()
+                for bp in bp_list:
+                    if bp_source_identifier == bp.file and \
+                       bp.line-1 < len(self.source) and bp.enabled:
+                        result.add(bp.line)
+                return result
+            old_bps = set_of_line_nums_from_bp_list(self._get_bp_list())
+            reload_breakpoints(w, size, key)
```

Comment:
Since I don't expect that `reload_breakpoints` uses its arguments, it doesn't make much sense to pass it bogus arguments. (same for this function, too). Simply remove the arguments, there and here. (and bind the key to a lambda that ignores the keys)